lib/controllers/ephemeral/setup_controller.dart

```diff
@@ -52,10 +34,10 @@ class SetupController extends GetxController {
       update();
       return;
     }
-    Get.find<CustomNavigator>().toNamed('/phone_number');
+    Get.find<AppNavigator>().toNamed('/phone_number');
   }
 
   void onLogin() {
-    Get.find<CustomNavigator>().offAllNamed('/dashboard');
+    Get.find<AppNavigator>().offAllNamed('/dashboard');
```

Comment:
I think instead of using a manually written String, we should use Routes.{screen}. I tried doing it before but Dart was throwing some error. 

Still, we should probably do it instead of manually writing out (which is prone to errors) the whole time.

Reply:
Yup, that's actually the reason I decided to keep the `AppNavigator`, but I think it would be better to perform the changes using another PR. I initially renamed the `CustomNavigator` to `AppNavigator` so that I could just do that, but stopped halfway to avoid making too many changes...

Do you perhaps want me to revert the renaming and just put all of them in another PR?

Reply:
No, I think just keep it as is for now, it would be some extra work - you are already done with your internship haha. I can look at doing that at some point in the future.

Reply:
Okay sure sounds good to me haha :-), thanks a lot!